File: lista1_js/44.latao.js
//44. Sabendo que latão é constituído de 70% de cobre e 30% de zinco, escreva um algoritmo que calcule a quantidade de cada um desses componentes para se obter certa quantidade de latão (em kg), informada pelo usuário.

import {question} from 'readline-sync'

//Entrada
const qtd_latao = Number(question('Informe a quantidade de latão: '))


//Processamento
const cobre = qtd_latao * 0.70
const zinco = qtd_latao * 0.30

//Saída
console.log(`\n>>> A quantidade de cobre é de ${cobre.toFixed(1)} e a de zinco é de ${zinco.toFixed(1)}, que compõem o latão.`)

File: lista1_py/44.latao.py
#44. Sabendo que latão é constituído de 70% de cobre e 30% de zinco, escreva um algoritmo que calcule a quantidade de cada um desses componentes para se obter certa quantidade de latão (em kg), informada pelo usuário.

#Entrada
qtd_latao = int(input('Informe a quantidade de latão: '))

#Processamento
cobre = qtd_latao * 0.70
zinco = qtd_latao * 0.30

#Saída
print(f'\n>>> A quantidade de cobre é de {cobre:.1f} e a de zinco é de {zinco:.1f}, que compõem o latão.')

